<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_editprof</class>
 <widget class="QDialog" name="fm_editprof">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Editar Professor</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Nome</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_nome_2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Telefone</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_telefone_2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>idade</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_idade_2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>email</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_email_2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>turma</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_turma_2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_15">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Salario</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_salario"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Confirmar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
